src/components/SchedulePage/Schedule.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import TimeSlot from '../SchedulePage/Timeslot';
+
+// Must be ordered
+const testEvents = [
+	{
+		name: 'ACM-W X ACM Hack | Hackathon 101 + Team Formation',
+		location: 'Sycamore Room',
+		// eslint-disable-next-line no-multi-str
+		description: 'First hackathon? We\'ll show you all the tips and tricks you\'ll ' +
+		'need to know to be successful and most importantly, have fun!',
+		startTime: '9:30 AM'
```

Comment:
I didn't know whether to use a `Date` here. As of now, times act like an arbitrary group, and the events are clumped together similar to this: 'aabaaacc' => 'abac'

Reply:
I think storing the times as `Date` objects to might be a good idea to make things easier down the road with the upcoming workshops section. Maybe we can do something like this:
``` javascript
const hothDate = '2021-02-23'; // replace with real date later
const utcOffset = '-08:00'; // PST

const testEvents = [
  {
  	title: "ACM does a backflip",
  	time: new Date(hothDate + 'T09:30:00' + utcOffset)
  }
];
```

I'm also wondering if perhaps it'd be a good idea to have the data structure arranged such that each object has a time and a list of events starting at that time? e.g,
``` javascript
const testEvents = [
  {
  	time: new Date(hothDate + 'T10:00:00' + utcOffset),
    events: [
      {
        name: 'Git and Github',
        location: 'De Neve Auditorium',
        // eslint-disable-next-line no-multi-str
        description: 'With multiple developers each working on ' +
        'a different feature of a project, bringing things together ' +
        'can get messy pretty easily. Come learn how to use Git to ' +
        'manage code across multiple collaborators and versions!',
        startTime: '10:00 AM'
      },
      {
        name: 'INTRO TO IOS',
        location: 'Sycamore Room',
        // eslint-disable-next-line no-multi-str
        description: 'Learn all about the fundamentals of iOS development ' +
        'by making a simple app in Swift and XCode! ' +
        'You must have a Mac to be able to code along in this workshop.',
        startTime: '10:00 AM'
      },
    ]
  },
  ...
];
```
This would circumvent the need to create the `(time, [events]))` pairs later on. What do you think?

Reply:
Yeah, I was unsure about how we wanted to handle the data. But the way you are preposing is one I considered. The only think is that formatting dates seems to be a bit of a pain in JS. There are some packages that help, but idk if it's worth adding another package.

Also, I considered grouping things be start time as well. I didn't do that because then it's really more of a list of "time slots" than a list of "events" which feels like it breaks an abstraction somewhere. However, if you think that that's a better way, I can do it that way. I was on the fence about a lot of it.

– sent from my iPhone while laying in bed